pretext/ExtraTopics/BigO.ptx

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-bigo">
+  <title>Big O Analysis</title>
+
+  <p>It's impossible to experimentally measure the theoretical performance of algorithms on a computer as the size of our data (which we call n) goes to infinity,
```

Comment:
This is a good start, but I'll probably want to re-do some of this myself rather than suggest changes through GitHub review.

Reply:
Sounds good.